tox.ini

```diff
@@ -2,7 +2,7 @@
 skipsdist = True
 usedevelop = True
 
-envlist = python{3.7,3.8}-django{2.2,3.0,3.1,master}-wagtail{2.11,2.12,2.13,2.14,main}-{sqlite,postgres}
+envlist = python{3.7,3.8}-django{2.2,3.0,3.1,master}-wagtail{2.11,2.12,2.13,2.14,2.15,main}-{sqlite,postgres}
```

Comment:
suggestion: add python 3.9 and Django 3.2 here and under basepython/deps below
suggestion: change `master` to `main` and make the corresponding change on line 27
suggestion: remove line 28 (where it declares djmaster as modelcluster)

Add

```
wagtail2.14: wagtail>=2.14,<2.15
wagtail2.15: wagtail>=2.15,<2.16
```

after `wagtail2.13: wagtail>=2.13,<2.14`